src/System.Net.NetworkInformation/src/System/Net/NetworkInformation/LinuxNetworkInterface.cs

```diff
@@ -68,15 +68,35 @@ private static LinuxNetworkInterface GetOrCreate(Dictionary<string, LinuxNetwork
             return lni;
         }
 
-        public override bool SupportsMulticast { get { return _supportsMulticast; } }
+        public override bool SupportsMulticast
+        {
+            get
+            {
+                if (_supportsMulticast.HasValue)
+                {
+                    return _speed.Value;
+                }
+                else
+                {
+                    throw new PlatformNotSupportedException(SR.net_InformationUnavailableOnPlatform);
```

Comment:
Should we just return false instead of throwing?  I realize there are three logical states: we know it's supported, we know it's not supported, and we don't know if it's supported.  I'm just wondering if there's a meaningful difference to a consumer between the last two.